src/rgw/rgw_dedup.h

```diff
@@ -0,0 +1,239 @@
+#pragma once
+#include "common/dout.h"
+#include "rgw_common.h"
+#include "rgw_dedup_utils.h"
+#include "rgw_dedup_table.h"
+#include "rgw_dedup_cluster.h"
+#include "rgw_realm_reloader.h"
+#include <string>
+#include <unordered_map>
+#include <variant>
+#include <iostream>
+#include <ostream>
+using namespace std;
```

Comment:
please avoid `using namespace std` in header files

Reply:
done